src/test/regress/expected/prepare_optimizer.out

```diff
@@ -164,6 +196,7 @@ SELECT name, statement, parameter_types FROM pg_prepared_statements
     ORDER BY name;
  name |                            statement                             |                  parameter_types                   
 ------+------------------------------------------------------------------+----------------------------------------------------
+ p    | PREPARE p AS SELECT * FROM generate_series(1, 10) i;             | {}
```

Comment:
Why not removing this statement after using not to make diff at the other place of the test?

Reply:
removed